src/lib/EventListener/AdminExceptionListener.php

```diff
@@ -45,6 +49,9 @@ class AdminExceptionListener
     /** @var string */
     protected $kernelEnvironment;
 
+    /** @var \Psr\Log\LogLevel::* */
+    protected $logLevel;
```

Comment:
Already exposed in constructor (as an argument). We don't expect extending classes (if any) to access it.
```suggestion
    private $logLevel;
```